### PR TITLE
[NUI] Provide APIs to access sub-components

### PR DESCRIPTION
**Pull request by zhouleonlei**
### Description of Change ###
(a) Provide APIs to access sub-components of the component (e.g. Button.ButtonIcon)

### CODE REVIEW

File: src/Tizen.NUI.Components/Controls/Slider.cs [hunk 1]
```diff
@@ -270,6 +270,86 @@ public enum IndicatorType
         /// <since_tizen> 8 </since_tizen>
         public new SliderStyle Style => ViewStyle as SliderStyle;
 
+        /// <summary>
+        /// Get or set background track.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public ImageView BgTrackImageBgTrackImage
```
[the remark this answers is not in the document]
Comment (zhouleonlei) [reply src/Tizen.NUI.Components/Controls/Slider.cs]: Fixed

[the remark this answers is not in the document]
Comment (zhouleonlei) [reply src/Tizen.NUI.Components/Controls/Slider.cs]: > Fixed



> How about BackgroundTrackImage?

Fixed

[the remark this answers is not in the document]
Comment (zhouleonlei) [reply src/Tizen.NUI.Components/Controls/Slider.cs]: > How about BackgroundTrackImage?

Fixed

File: src/Tizen.NUI.Components/Controls/DropDown.cs [hunk 2]
```diff
@@ -205,8 +205,125 @@ public enum ListOrientation
...
+                        ParentOrigin = NUI.ParentOrigin.Center,
+                        PivotPoint = NUI.ParentOrigin.Center,
+                        PositionUsesPivotPoint = true,
+                        Name = "DropDownHeaderText"
+                    };
+                    Add(headerText);
+                }
+                return headerText;
+            }
+            set
```
Comment (rabbitfor) [inline src/Tizen.NUI.Components/Controls/DropDown.cs:234]: internal set

Comment (rabbitfor) [reply src/Tizen.NUI.Components/Controls/DropDown.cs:234]: Can you make all sub-components setters to internal?

File: src/Tizen.NUI.Components/Controls/DropDown.cs [hunk 3]
```diff
@@ -384,18 +501,18 @@ public override void ApplyStyle(ViewStyle viewStyle)
             DropDownStyle dropDownStyle = viewStyle as DropDownStyle;
             if (null != dropDownStyle)
             {
-                CreateHeaderText();
-                CreateButtonText();
-                CreateButton();
-
-                CreateListBackgroundImage();
```
Comment (rabbitfor) [inline src/Tizen.NUI.Components/Controls/DropDown.cs:391]: Please remove `private void CreateListBackgroundImage` function because it is using now.

Comment (rabbitfor) [reply src/Tizen.NUI.Components/Controls/DropDown.cs:391]: Same as `private void CreateHeaderText()`, `private void CreateButtonText()` and `private void CreateButton()`

**Comment by rabbitfor** [comment]
In my opinion, it would be better not to release the APIs below for now.
* Slider.BackgroundTrackImage
* Slider.SlidedTrackImage
* Slider.ThumbImage
* Toast.TextLabel

And we are considering to refactor scrollbar APIs (some of work are done now),
so can you revert `src/Tizen.NUI.Components/Controls/Scrollbar.cs`?
And thank you if you rebase it with latest TizenFX after then.

File: src/Tizen.NUI.Components/Controls/DropDown.cs (hunk 2 above)
Comment (zhouleonlei) [reply src/Tizen.NUI.Components/Controls/DropDown.cs]: > internal set

Fixed

**Review by AdunFang — COMMENTED** [review]
It is OK.

**Pull request merged**